java-contrib.yaml

```diff
@@ -0,0 +1,75 @@
+package:
+  name: java-contrib
```

Comment:
The package name should probably be something like `opentelemetry-java-contrib` to make its purpose clear. Otherwise I'd read it as being something more generic for the java ecosystem.

Reply:
@jamonation Very long name but renamed all packages :-)